api/src/pcapi/core/offers/validation.py

```diff
@@ -228,6 +230,12 @@ def check_image(
     if image.width < min_width or image.height < min_height:
         raise exceptions.ImageTooSmall(min_width, min_height)
 
+    if max_width is not None and max_height is not None and image.width > max_width:
```

Comment:
Pour une compréhension plus rapide (gagner 5 secondes :)), les deux conditions ne pourraient-elles pas se simplifier en cette ligne ?
`if max_width is not None and max_height is not None and (image.width > max_width or image.height > max_height):`

Reply:
j'ai reécrit l'exception pour que les conditions soient plus simples.

Reply:
la comparaison en hauteur et en largeur doivnt rester indépendantes (les paramatre étant facultatifs)